Makefile.work

```diff
@@ -242,3 +242,7 @@ reset :
 	    else \
 	        echo "Reset aborted"; \
 	    fi
+ifneq (,$(filter $(CONFIGURED_ARCH), armhf arm64))
```

Comment:
Should this not be placed inside the `if` statement? Otherwise, if someone decides to abort the operation, these commands will still be executed.

Reply:
I have addressed the comment, having inside if "y". 
I had to do cleanup on reset target by adding .ONESHELL, as a cleanup  for "if/else" conditions